open-machine-learning-jupyter-book/assignments/set-up-env/first-assignment.ipynb

```diff
@@ -24,22 +24,22 @@
   },
```

Comment:
<p>Such code is not your change. Please exclude them from the PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Did you resolve this?</p>